### PR TITLE
Add data for the new `options.min` parameter of `ReadableStreamBYOBReader.read()`

### PR DESCRIPTION
**Pull request by skyclouds2001**
#### Summary

see https://github.com/whatwg/streams/pull/1145

the feature has shipped in [node](https://github.com/nodejs/node/issues/50706), and done in [deno](https://github.com/denoland/deno/pull/20849) though seems to be not released, also, [firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1864406) has implemented it and seems to be shipped in M134

also see https://github.com/mdn/mdn/issues/478

### CODE REVIEW

File: api/ReadableStreamBYOBReader.json [hunk 1]
```diff
@@ -213,6 +213,55 @@
...
+                "version_added": false,
+                "impl_url": "https://crbug.com/40942083"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": false,
```
Comment (caugner) [inline api/ReadableStreamBYOBReader.json:231]: Based on the resolved bug, it's in Nightly:

```suggestion
                "version_added": "preview",
```

File: api/ReadableStreamBYOBReader.json [hunk 2]
```diff
@@ -213,6 +213,55 @@
             "standard_track": true,
             "deprecated": false
           }
+        },
+        "options_min_parameter": {
+          "__compat": {
+            "description": "<code>options.min</code> parameter",
```
Comment (caugner) [inline api/ReadableStreamBYOBReader.json:219]: We could try an `impl_url` with a text fragment here:

```suggestion
            "description": "<code>options.min</code> parameter",
            "impl_url: "https://streams.spec.whatwg.org/#:~:text={%20min%20}",
```

File: browsers/nodejs.json [hunk 3]
```diff
@@ -594,6 +601,13 @@
           "engine": "V8",
           "engine_version": "11.8"
         },
+        "21.7.0": {
+          "release_date": "2024-05-06",
```
Comment (caugner) [inline browsers/nodejs.json:605]:
```suggestion
          "release_date": "2024-03-06",
```

**Review by caugner — CHANGES_REQUESTED** [review]

File: api/ReadableStreamBYOBReader.json [hunk 4]
```diff
@@ -213,6 +213,56 @@
...
+            "impl_url: "https://streams.spec.whatwg.org/#:~:text={%20min%20}",
```
Comment (skyclouds2001) [inline api/ReadableStreamBYOBReader.json:220]:
```suggestion
            "impl_url: "https://streams.spec.whatwg.org/#dom-readablestreambyobreaderreadoptions-min",
```

File: api/ReadableStreamBYOBReader.json [hunk 5]
```diff
@@ -213,6 +213,56 @@
...
+                "impl_url": "https://crbug.com/40942083"
+              },
+              "chrome_android": "mirror",
+              "deno": {
+                "version_added": false
+              },
+              "edge": "mirror",
+              "firefox": {
+                "version_added": "preview",
+                "impl_url": "https://bugzil.la/1864406"
```
Comment (skyclouds2001) [inline api/ReadableStreamBYOBReader.json:233]:
```suggestion
                "version_added": "preview"
```

**Comment by isker** [comment]
> and done in https://github.com/denoland/deno/pull/20849 though seems to be not released

Why do you say that?  Looks like it's been released for a long time: https://github.com/denoland/deno/releases/tag/v1.39.0

**Comment by skyclouds2001** [comment]
> > and done in [denoland/deno#20849](https://github.com/denoland/deno/pull/20849) though seems to be not released
> 
> Why do you say that? Looks like it's been released for a long time: https://github.com/denoland/deno/releases/tag/v1.39.0

Good report!

**Pull request merged**